File: cpp/quadrante.cpp
#include <iostream>

using namespace std;

int main(){
    int x, y;

    cout << "Digite os valores das coordenadas X e Y:" << endl;
    cin >> x >> y;

    while(x != 0 && y != 0){
        if(x > 0 && y > 0){
            cout << "QUADRANTE Q1" << endl;
        }
        else if(x < 0 && y > 0){
            cout << "QUADRANTE Q2" << endl;
        }
        else if(x < 0 && y < 0){
            cout << "QUADRANTE Q3" << endl;
        }
        else{
            cout << "QUADRANTE Q4" << endl;
        }

        cout << "Digite os valores das coordenadas X e Y:" << endl;
        cin >> x >> y;
    }

	return 0;
}

File: python/quadrante.py
x: int; y: int

print("Digite os valores das coordenadas X e Y:")
x = int(input())
y = int(input())

while x != 0 and y != 0:
	if x > 0 and y > 0:
		print("QUADRANTE Q1")
	elif x < 0 and y > 0:
		print("QUADRANTE Q2")
	elif x < 0 and y < 0:
		print("QUADRANTE Q3")
	else:
		print("QUADRANTE Q4")

	print("Digite os valores das coordenadas X e Y:")
	x = int(input())
	y = int(input())
